inc/api.class.php

```diff
@@ -1083,14 +1083,14 @@ protected function getItems($itemtype, $params = [], &$totalcount = 0) {
          foreach ($params['searchText']  as $filter_field => $filter_value) {
             if (!empty($filter_value)) {
                $search = Search::makeTextSearch($filter_value);
-               $where.= " AND (`$table`.`$filter_field` $search
-                               OR `$table`.`id` $search)";
+               $where.= " AND (`$table`.`$filter_field` $search)";
```

Comment:
I really do not know if this is ok :/

Reply:
It's idiot, if we search `tickettemplates_id` = 1, it will get all right line + the line with `id` =1, so the list return is wrong because we don't want the `id=1`